libcxx/include/__type_traits/is_allocator.h

```diff
@@ -21,13 +21,21 @@
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
+#if _LIBCPP_STD_VER >= 20
```

Comment:
Why?

Reply:
Because the variable template change is (IMO) a pure improvement, while adding a branch increases the complexity of the implementation. IMO this deserves it's own conversation and some evidence that this actually improves diagnostics. The variable template change is also fully self-contained. Making the concept change a follow-up patch makes it much smaller and more focused.